flyway-core/src/main/java/org/flywaydb/core/internal/database/postgresql/PostgreSQLConnection.java

```diff
@@ -31,7 +31,7 @@
 public class PostgreSQLConnection extends Connection<PostgreSQLDatabase> {
     private final String originalRole;
 
-    PostgreSQLConnection(PostgreSQLDatabase database, java.sql.Connection connection) {
+    protected PostgreSQLConnection(PostgreSQLDatabase database, java.sql.Connection connection) {
```

Comment:
What has Cockroach done? Haven't they extended or they have fully implemented the apis? I am not sure if Flyway will be ready to let us modify the Postgres flyway classes.

Reply:
CockroachDB does not extend the PostgreSQL classes.